src/main/java/core/basesyntax/ProductOperation.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public enum ProductOperation {
+    SUPPLY("supply"),
+    BUY("buy"),
+    RESULT("result");
+
+    private final String value;
+
+    ProductOperation(String value) {
+        this.value = value;
+    }
+
+    static String getValue(ProductOperation op) {
+        return op.value;
+    }
```

Comment:
We can work with enum attributes the same way as we work with class attributes, just retrieving the ```value``` of enum entry itself
Like
```suggestion
    public String getValue() {
        return value;
    }
```
And use it as
```
SUPPLY.getValue()
```
which is a bit simpler than
```
ProductOperation.getValue(SUPPLY)
```

Reply:
Thanks for your wonderful suggestion!